tests/migrate/test_subtasks.py

```diff
@@ -0,0 +1,63 @@
+# pylint: disable-all
+
+from recommender.commands.migrate import _apply, _rollback
+from recommender.models.migration import Migration
+
+
+def test__apply_happy_path(mongo, mock_migrations_dir):
+    _apply()
+    migration_objects = Migration.objects().order_by("+name")
+
+    assert len(migration_objects) == 2
+
+    for m in migration_objects:
+        assert m.applied
+
+    _apply()
+    assert len(migration_objects) == 2
+
+    for m in migration_objects:
+        assert m.applied
+
+
+def test__apply_integrity_check_fail(mongo, mock_migrations_dir):
+    Migration(name="20211203092545_missing_migration", applied=False).save()
+    _apply()
+
+    migration_objects = Migration.objects().order_by("+name")
+    assert len(migration_objects) == 3
+    for m in migration_objects:
+        assert not m.applied
+
+
+def test__rollback_happy_path(mongo, mock_migrations_dir):
+    _apply()
+
+    migration_objects = Migration.objects().order_by("+name")
+
+    assert migration_objects[0].applied
+    assert migration_objects[1].applied
+
+    _rollback()
+
+    assert migration_objects[0].applied
+    assert not migration_objects[1].applied
+
+    _rollback()
+
+    assert not migration_objects[0].applied
+    assert not migration_objects[0].applied
+
+
+def test__rollback_integrity_check_fail(mongo, mock_migrations_dir):
```

Comment:
Here we have 3 applied migrations. Then we do a single `rollback`.
Why the expected behaviour here is to have all 3 migrations with the status `applied`?